kernel/core/vfs/fs_jail.c

```
@@ -0,0 +1,92 @@
+#include <unistd.h>
+
+#define JAIL_ROOT_MAX 4096
+static char   g_fs_jail_root[JAIL_ROOT_MAX];
+static size_t g_fs_jail_len;
+
+void fs_jail_init(void) {
+    mem_domain_zero(g_fs_jail_root, sizeof(g_fs_jail_root));
+    g_fs_jail_len = 0;
+    if (!g_vm_mode)
+        return;
+    char wd[JAIL_ROOT_MAX];
```

Clang (14.0.6)

[warning] variable name 'wd' is too short, expected at least 3 characters

(readability-identifier-length)